src/input/mpegts/satip/satip.c

```diff
@@ -384,12 +398,6 @@ satip_device_hack( satip_device_t *sd )
     sd->sd_pids_max    = 128;
     sd->sd_pids_len    = 2048;
     sd->sd_no_univ_lnb = 1;
-  } else if (strstr(sd->sd_info.manufacturer, "AVM Berlin") &&
```

Comment:
OK. I see your point (I thought that the hacks are applied before the settings is loaded, but it's after this operation). I prefer to change the code in this case (when the firmware is fixed). This settings is really for advanced users and I prefer to check the firmware capabilities manually. But for "unknown" firmware, all settings should be available. Perhaps, we can add an option 'do not apply any automatic hacks' rather..